### PR TITLE
Wall, Airlocks and Tile resprites

### PR DESCRIPTION
**Pull request by noctyrnal**
## About the PR

Resprites the basic walls, most airlocks and tiles and most deals, along with adding two new sets of decals. I intent on doing more PRs in the future to resprite more content.

## Why / Balance

Provides Delta-V with a unique visual identity originating from SS13 forks with content either being fully ported, edited and ported, or wholly created by myself to match the style created with this PR.

## Media

ed73d6049e360b9c9a8619967494b0ea
0d726a3de498a89d19ff8a328b5fc872
6a9c9634ddeb76c546fd8108ac71c355
eb5944aeae6d60b0ff6acdf531b3d193

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Walls, Airlocks and Tiles have received new sprites as per a visual resprite project.

### CODE REVIEW

**Comment by noctyrnal** [comment]
So many files changed holy shit let me go kill this PR and cut it into pices

**Comment by ghost** [comment]
Noctyrn my brother in contribution, I'm reopening this

**Comment by noctyrnal** [comment]
> Noctyrn my brother in contribution, I'm reopening this

I was going to do at a later date, need to make this less of a nightmare for maintainers to handle

**Comment by noctyrnal** [comment]
Marked as draft until we have more people to maintain a project of this size o7

**Review by WarMechanic — COMMENTED** [review]
I hate to be the 'more work' guy but I'm going to suggest earlier rather than later:

RCD prototypes have a sprite specifier that determines their ui icon. Currently these point to a unique icon sprite - in the case of the airlock structure its the same sprite but nearest-neighbour squashed to add 5 pixel padding on the edges. You could either emulate this, or have the icons point to the new DeltaV sprites directly.

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml
Comment (Axionyxx) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml:13]: These are in our namespace already, and as such don't need comments

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml
Comment (Axionyxx) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml:19]: Same here

**Review by Axionyxx — DISMISSED** [review]
Besides these 2, everything looks fine! I trust the automated tests for the rest.

Adei told me I wouldn't do it so I reviewed this. I looked at like 3 or 4 of the meta.json's, and they all seem fine.

**Comment by Axionyxx** [comment]
If it turns out to be too much to maintain, we can always revert

**Review by Axionyxx — DISMISSED** [review]
Reapproving :Godo:

**Comment by Axionyxx** [comment]
@noctyrnal before I merge this, was the only reason for the draft maintaining concerns?

**Comment by noctyrnal** [comment]
> @noctyrnal before I merge this, was the only reason for the draft maintaining concerns?

yeah, since it's an absolutely massive pr and I imagine just lil old me couldnt maintain it solo

**Comment by Axionyxx** [comment]
> > @noctyrnal before I merge this, was the only reason for the draft maintaining concerns?
> 
> yeah, since it's an absolutely massive pr and I imagine just lil old me couldnt maintain it solo

It isn't that massive when you disregard sprites. They don't add new wall textures every day and when they do, they don't tend to suddenly be mapped

**Comment by noctyrnal** [comment]
> > > @noctyrnal before I merge this, was the only reason for the draft maintaining concerns?
> > 
> > 
> > yeah, since it's an absolutely massive pr and I imagine just lil old me couldnt maintain it solo
> 
> It isn't that massive when you disregard sprites. They don't add new wall textures every day and when they do, they don't tend to suddenly be mapped

I concur, looking forward to continuing my work in the future hopefully in smaller PRs

**Review by ghost — APPROVED** [review]
Looking this PR from top to bottom, I think it's pretty good and safe to merge, with the changes made that null proposed. Sheer size (900 files) is pretty big, but remember - if we can't maintain it we can always revert.

**Review by Axionyxx — APPROVED** [review]
"Do it no balls"
~ Adei

**Comment by noctyrnal** [comment]
Thank you Adei and Null, this is truly a high command sweep.

**Comment by Verbalase** [comment]
it's all coming along... slowly every server will get it

overall hooray

**Comment by NakataRin** [comment]
I guess my opinion doesn't matter much, if at all, but I still hope for a miracle 
This change makes delta-v look like the majority of russian servers. I fully disapprove this change

**Pull request merged**
